src/main/scala/scalafix/internal/sbt/ScalafixInterface.scala

```diff
@@ -116,47 +116,60 @@ object ScalafixInterface {
     private lazy val _value = scala.util.Try(thunk())
     override def apply(): T = _value.get
   }
+  private val fromToolClasspathMemo: scala.collection.mutable.HashMap[
```

Comment:
This instance will persist after a `sbt reload`, so we might accumulate stale `ScalafixInterface` instances if the user keeps changing the build definition in a way that impacts the composite key below. The alternative might be to keep that cache in a build-level setting key, but I am not even sure the classloader would reclaim everything when the old cache/setting value gets garbage-collected after a `sbt reload`? :thinking: 

Let's keep this as-is considering the cardinality is very low :+1: